test/spec/tasks/check_enabled_tests.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Mix.Tasks.CheckEnabledTests do
+  @moduledoc """
+  A task that checks which spec-tests are enabled or skipped. Useful to make sure that desired
+  tests are running.
+
+  It traverses vector configs, forks and runners hierarchically.
+
+  Run with MIX_ENV=test mix check_enabled_tests
+  The output will be in a reports/spec_test_report.exs file, that can be formatted using mix format.
+  """
+
+  require Logger
+
+  alias Spec.MetaUtils
+  @output_dir "reports"
+
+  def run(_args) do
+    # TODO: pretty print.
+    res = MetaUtils.check_enabled() |> inspect(limit: :infinity)
+    File.mkdir_p!(@output_dir)
+    File.write!(filename(), res)
```

Comment:
I think we should just `IO.puts` this, for simplicity.

Reply:
Removed the file, added pretty printing.